### DISCUSSION TITLE
TEG: a simple Reinforcement learning environment to train a robot arm

### DISCUSSION THREAD

**Discussion by alefram** [google-deepmind/mujoco#899] [Show and tell]
Hello! I would like to share my bachelor's thesis which involves the application of reinforcement learning in robotics. For this project, I utilized the Mujoco Python library and Gym to create it. I am planning to further develop it by incorporating more environments and tools.🤓

https://github.com/alefram/TEG

**Comment by kevinzakka** [comment]
Thanks for sharing :)
